utils/auth/google_authorizer.go

```diff
@@ -0,0 +1,128 @@
+package auth
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/labstack/echo"
+	"github.com/satori/go.uuid"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+)
+
+const googleInfoEndpoint = "https://www.googleapis.com/oauth2/v3/userinfo"
+const uclaDomain = "g.ucla.edu"
```

Comment:
does that cover all subdomains? cs.ucla.edu, biology.ucla.edu etc.

Reply:
Hard call, can’t really test it because we don’t have an account to test with